pkg/k8s/pods/pod.go

```diff
@@ -27,19 +28,20 @@ const (
 	// OktetoSyncLabel indicates a synthing pod
 	OktetoSyncLabel = "syncthing.okteto.com"
 
-	maxRetriesPodRunning = 1500 //5min pod is running
+	maxRetriesPodRunning = 300 //1min pod is created
 )
 
 var (
 	devTerminationGracePeriodSeconds int64
 )
 
 // GetDevPod returns the dev pod for a deployment
-func GetDevPod(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset, waitUntilDeployed bool) (*apiv1.Pod, error) {
+func GetDevPod(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset, waitUntilDeployed bool, reporter chan string) (*apiv1.Pod, error) {
+	defer close(reporter)
 	tries := 0
 	ticker := time.NewTicker(200 * time.Millisecond)
 	for tries < maxRetriesPodRunning {
```

Comment:
if we have a watcher pattern in `loopGetDevPod`, I don't think we should have the retries. This would complicate consuming the `reporter`.  Could we simplify everything on a single model?  Get the pod, watch until ready, return?